tests/test_vec/conftest.py

```diff
@@ -14,3 +14,21 @@ def data():
         for line in f.readlines():
             _data.append(json.loads(line))
     return _data
+
+
+@pytest.fixture(scope="module")
+def disen_train_data():
+    _data = []
+    with codecs.open(path_append(abs_current_dir(__file__), "disenq_train.json", to_str=True), encoding="utf-8") as f:
```

Comment:
I think maybe it is better to move all the data files for testing to somewhere else (e.g. `tests/test_vec/data/`). It looks a little messy leaving all the data in the same path with code.

Reply:
Yes, i will deal with it